sound/soc/sof/sof-ipc-test.c

```diff
@@ -7,13 +7,40 @@
 #include <linux/module.h>
 #include <linux/virtual_bus.h>
 #include <linux/mod_devicetable.h>
+#include "sof-client.h"
 
-static int sof_ipc_test_probe(struct virtbus_device *vdev)
+static int sof_ipc_client_probe(struct sof_client_dev *cdev)
 {
-	dev_dbg(&vdev->dev, "probing %s\n", vdev->name);
+	dev_dbg(cdev->dev, "SOF client probing\n");
 	return 0;
 }
 
+static void sof_ipc_client_remove(struct sof_client_dev *cdev)
+{}
```

Comment:
you didn't put a "mandatory" comment next to the `.remove` operation, isn't it optional then?

Reply:
ahh yes. I got confused there. the remoe is mandatory for the virtbus device but for the SOF client device. Will remove it.